### ISSUE TITLE
[bug report]java.lang.IllegalArgumentException: URL is malformed

### ISSUE THREAD

**Issue by rxxy** [MCMicS/jenkins-control-plugin#440]
```
java.lang.IllegalArgumentException: URL is malformed
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.handleException(UrlBuilder.java:158)
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.buildUrl(UrlBuilder.java:219)
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.createFillValueItemsUrl(UrlBuilder.java:212)
	at org.codinjutsu.tools.jenkins.logic.RequestManager.getFillValueItems(RequestManager.java:506)
	at org.codinjutsu.tools.jenkins.logic.RequestManager.lambda$getGitParameterChoices$11(RequestManager.java:500)
	at java.base/java.util.Optional.map(Optional.java:260)
	at org.codinjutsu.tools.jenkins.logic.RequestManager.getGitParameterChoices(RequestManager.java:500)
	at org.codinjutsu.tools.jenkins.view.extension.JobParameterRenderers.createGitParameterChoices(JobParameterRenderers.java:163)
	at org.codinjutsu.tools.jenkins.view.extension.JobParameterRenderers.lambda$createGitParameterChoices$2(JobParameterRenderers.java:149)
	at org.codinjutsu.tools.jenkins.view.parameter.GitParameterRenderer.render(GitParameterRenderer.java:56)
	at org.codinjutsu.tools.jenkins.view.BuildParamDialog.addParameterInputs(BuildParamDialog.java:119)
	at org.codinjutsu.tools.jenkins.view.BuildParamDialog.<init>(BuildParamDialog.java:71)
	at org.codinjutsu.tools.jenkins.view.BuildParamDialog.lambda$showDialog$0(BuildParamDialog.java:79)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:838)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:454)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:74)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:114)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:909)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:756)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$5(IdeEventQueue.java:437)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:436)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:615)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:434)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:838)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:480)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.net.MalformedURLException: no protocol: descriptorByName/net.uaznia.lukanus.hudson.plugins.gitparameter.GitParameterDefinition/fillValueItems?param=BRANCH_TAGdescriptorByName/net.uaznia.lukanus.hudson.plugins.gitparameter.GitParameterDefinition/fillValueItems?param=BRANCH_TAG
	at java.base/java.net.URL.<init>(URL.java:674)
	at java.base/java.net.URL.<init>(URL.java:569)
	at java.base/java.net.URL.<init>(URL.java:516)
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.buildUrlNotNull(UrlBuilder.java:227)
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.buildUrl(UrlBuilder.java:217)
	... 42 more

```

**Comment by rxxy** [comment]
![image](https://user-images.githubusercontent.com/14834156/215642698-af56815e-d353-4db5-b6c6-54cff87071d6.png)

![image](https://user-images.githubusercontent.com/14834156/215642855-8e5b2d9d-09c6-4484-94a3-f32ded388786.png)

**Comment by MCMicS** [comment]
Thanks for filling the issue.
i will check and fix it

**Comment by Maoway** [comment]
Our team also encountered this problem

**Comment by MCMicS** [comment]
I try to reproduce it

it seems that this occurs during get parameter for Git parameters.

Is this right?

Do you want to run a build with parameters?

**Comment by rxxy** [comment]
![image](https://user-images.githubusercontent.com/14834156/215686917-81e322a9-4636-4015-ab0d-5c40a7435282.png)
Whenever I click on [build on Jenkins] this error happens, no popup as expected

**Comment by MCMicS** [comment]
Ok but the job has a git parameter?

**Comment by rxxy** [comment]
Yes，In addition to branch_tag, there is also an enriroment parameter

**Comment by rxxy** [comment]
![image](https://user-images.githubusercontent.com/14834156/215689451-f33314cb-5bc0-4754-be46-3a3f6b330e31.png)

**Comment by MCMicS** [comment]
Thanks
i try to reproduce it. Or is your server public available. You can also send me privately on gitter if you like

**Comment by MCMicS** [comment]
I find to bug. I fix it

**Comment by MCMicS** [comment]
Please use following build: https://plugins.jetbrains.com/plugin/6110-jenkins-control/versions/eap/285114

Or add EAP as plugin site to update from IDE

see: https://github.com/MCMicS/jenkins-control-plugin#use-eap-builds

**Comment by rxxy** [comment]
It works now! awesome

**Comment by Maoway** [comment]
It works now! Thanks

**Comment by MCMicS** [comment]
nice to hear
sorry for the bug

**Comment by MCMicS** [comment]
released with [0.13.17](../releases/tag/v0.13.17)
